frontend/src/components/UserComponents/UserCard.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import PropTypes from "prop-types";
+import axios from "axios";
+
+function UserCard({ user, setIsUpdated }) {
+  const handleDelete = () => {
+    axios
+      .delete(`${import.meta.env.VITE_BACKEND_URL}/api/user/${user.id}`)
+      .then(() => {
+        setIsUpdated(true);
+      })
+      .catch((e) => console.error(e));
+  };
+
+  return (
+    <div className="bg-white bg-opacity-60 rounded-xl mb-4 p-3 shadow-md flex flex-col justify-center">
+      <div>
+        <strong>Joueur:</strong> {user.username}
```

Comment:
bad usage of <strong> here. Check the doc : https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong

Reply:
Done !